static/app/components/replays/diff/replaySideBySideImageDiff.tsx

```diff
@@ -28,31 +29,24 @@ export function ReplaySideBySideImageDiff({leftOffsetMs, replay, rightOffsetMs}:
       </DiffHeader>
 
       <ReplayGrid>
-        <ReplayContextProvider
-          analyticsContext="replay_comparison_modal_left"
-          initialTimeOffsetMs={{offsetMs: leftOffsetMs}}
-          isFetching={fetching}
-          replay={replay}
-        >
-          <Border>
-            <ReplayPlayer isPreview />
-          </Border>
-        </ReplayContextProvider>
-
-        <ReplayContextProvider
-          analyticsContext="replay_comparison_modal_right"
-          initialTimeOffsetMs={{offsetMs: rightOffsetMs}}
-          isFetching={fetching}
-          replay={replay}
-        >
-          {rightOffsetMs > 0 ? (
+        <ReplayPlayerPluginsContextProvider>
+          <ReplayPlayerEventsContextProvider replay={replay}>
+            <Border>
+              <ReplayPlayerStateContextProvider>
+                <ReplayPlayerMeasurer measure="width">
+                  {style => <ReplayPlayer style={style} offsetMs={leftOffsetMs} />}
+                </ReplayPlayerMeasurer>
+              </ReplayPlayerStateContextProvider>
+            </Border>
             <Border>
-              <ReplayPlayer isPreview />
+              <ReplayPlayerStateContextProvider>
+                <ReplayPlayerMeasurer measure="both">
```

Comment:
above we've got `measure="width"`
idk which is best, i forgot what's what. but i expected them to be the same.

Reply:
oops, it's fixed, should be `width`